tools/swift-inspect/build_script_helper.py

```diff
@@ -49,7 +49,7 @@ def main():
     if not os.path.isdir(bin_dir):
         os.makedirs(bin_dir)
 
-    swiftbuild_path = os.path.join(args.toolchain, "usr", "bin", "swift-build")
```

Comment:
Nice, I removed several of these in #30565 and related pulls a couple years ago, but I don't build these products, so it looked like these slipped through.